source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -247,6 +247,14 @@ def getContainerCpuMetricItems(metricJSON, hostName, cpuMetricNameToCollect, met
               metricItem["json_Collections"] = metricCollections.to_json
               metricItems.push(metricItem)
 
+              if metricValue > 10000000000
```

Comment:
probable we can just log the exception with container name and value if the cpu usage value more than node's cpu usage, right?

Reply:
I was thinking that we shouldn't rely on the numbers coming from the cadvisor endpoint. Currently the value is 10^10.
Do you think a number like this makes more sense or the node cpu usage value? I will update it accordingly.